Task6/Task6.Data/Repositories/Abstraction/IRepository.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Task6.Data.Repositories.Abstraction
+{
```

Comment:
You have defined the interface without methods, and not use it at all, what for?